ext/standard/math.c

```diff
@@ -1296,6 +1296,24 @@ PHP_FUNCTION(fmod)
 }
 /* }}} */
 
+/* {{{ proto float fdiv(float x, float y)
```

Comment:
Recommend this to be documented as `proto float fdiv(float dividend, float divisor)` -- aligns with `intdiv` naming and clarifies which is numerator and which is denominator.

Reply:
Done!